sympy/matrices/dense.py

```diff
@@ -290,6 +305,126 @@ def rot_axis1(theta):
     return Matrix(lil)
 
 
+def rot_ccw_axis3(theta):
+    """Returns a rotation matrix for a rotation of theta (in radians)
+    about the 3-axis.
+
+    For a right-handed coordinate system, this corresponds to a
+    counterclockwise rotation.
```

Comment:
It could be better to explicitly leave the definition as mathematical formula under `Explanation` 
section because it is more clear though.

Reply:
As the exact formula of the matrix?